{"cveId": "CVE-2021-22239", "cwe": ["CWE-863", "CWE-863"], "cvss": [{"baseScore": "5.0", "version": "3.1", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "Low", "User Interaction": "None", "Scope": "Complete", "Confidentiality Impact": "None", "Integrity Impact": "Low", "Availability Impact": "None"}], "references": ["https://gitlab.com/gitlab-org/gitlab/-/issues/336301", "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22239.json"], "description": ["An unauthorized user was able to insert metadata when creating new issue on GitLab CE/EE 14.0 and later."], "published": "2021-09-09T15:15:08.683", "state": "PUBLIC", "vendorName": ["GitLab"], "productName": ["GitLab"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}